ghost/core/core/server/services/email-analytics/lib/queries.js

```diff
@@ -41,10 +41,14 @@ module.exports = {
     },
 
     async aggregateEmailStats(emailId) {
+        const [deliveredCount] = await db.knex('email_recipients').count('id as count').whereRaw('email_id = ? AND delivered_at IS NOT NULL', [emailId]);
```

Comment:
Would it be more efficient to do these using `Promise.all`?

Reply:
Possibly but wouldn't that add extra load on the db at this point which is something we're trying to avoid?

Reply:
On Ghost's side we're not hugely worried about overall runtime here.